cli/config.toml

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Do we need both `cli/config.toml` and `cli/template.config.toml` ?

Reply:
@SammyOina requested that we rename cli/template to cli/config
Let me remove template